packages/kyt-core/src/config/webpackHotDevClient.js

```diff
@@ -0,0 +1,273 @@
+// This alternative WebpackDevServer combines the functionality of:
+// https://github.com/webpack/webpack-dev-server/blob/webpack-1/client/index.js
+// https://github.com/webpack/webpack/blob/webpack-1/hot/dev-server.js
+
+// It only supports their simplest configuration (hot updates on same server).
+// It makes some opinionated choices on top, like adding a syntax error overlay
+// that looks similar to our console output. The error overlay is inspired by:
+// https://github.com/glenjamin/webpack-hot-middleware
+
+/* eslint-disable no-use-before-define */
+
+const SockJS = require('sockjs-client');
+const stripAnsi = require('strip-ansi');
+const url = require('url');
+const launchEditorEndpoint = require('react-dev-utils/launchEditorEndpoint');
+const formatWebpackMessages = require('react-dev-utils/formatWebpackMessages');
+const ErrorOverlay = require('react-error-overlay');
+
+const sockJSPort = KYT.CLIENT_PORT || parseInt(window.location.port || 3000, 10) + 1;
```

Comment:
Not sure how important this is, but `KYT.CLIENT_PORT` will be a String, so this will be a String or Number depending on whether that's defined

Reply:
just pushed a parseInt() for this